### ISSUE TITLE
cannot open project with latest CheckStyle IDEA plugin

### ISSUE THREAD

**Issue by todzheng** [jshiell/checkstyle-idea#143]
2015-04-08 08:53:30,169 [   9018]   INFO - .intellij.idea.IdeaApplication - App initialization took 10670 ms 
2015-04-08 08:53:37,790 [  16639]   INFO - roject.impl.ProjectManagerImpl - com.intellij.diagnostic.PluginException: org/infernus/idea/checkstyle/CheckStylePlugin : Unsupported major.minor version 52.0 [Plugin: CheckStyle-IDEA] [Plugin: CheckStyle-IDEA] 
com.intellij.ide.plugins.PluginManager$StartupAbortedException: com.intellij.diagnostic.PluginException: org/infernus/idea/checkstyle/CheckStylePlugin : Unsupported major.minor version 52.0 [Plugin: CheckStyle-IDEA] [Plugin: CheckStyle-IDEA]
        at com.intellij.ide.plugins.PluginManager.handleComponentError(PluginManager.java:245)
        at com.intellij.openapi.components.impl.PlatformComponentManagerImpl.handleInitComponentError(PlatformComponentManagerImpl.java:39)
        at com.intellij.openapi.components.impl.ComponentManagerImpl$ComponentsRegistry.a(ComponentManagerImpl.java:441)
        at com.intellij.openapi.components.impl.ComponentManagerImpl$ComponentsRegistry.a(ComponentManagerImpl.java:417)
        at com.intellij.openapi.components.impl.ComponentManagerImpl$ComponentsRegistry.access$000(ComponentManagerImpl.java:403)
        at com.intellij.openapi.components.impl.ComponentManagerImpl.a(ComponentManagerImpl.java:108)
        at com.intellij.openapi.components.impl.ComponentManagerImpl.init(ComponentManagerImpl.java:87)
        at com.intellij.openapi.project.impl.ProjectImpl.init(ProjectImpl.java:295)
        at com.intellij.openapi.project.impl.ProjectManagerImpl.a(ProjectManagerImpl.java:319)
        at com.intellij.openapi.project.impl.ProjectManagerImpl.access$700(ProjectManagerImpl.java:82)
        at com.intellij.openapi.project.impl.ProjectManagerImpl$12.compute(ProjectManagerImpl.java:629)
        at com.intellij.openapi.project.impl.ProjectManagerImpl$12.compute(ProjectManagerImpl.java:625)
        at com.intellij.openapi.progress.impl.CoreProgressManager$4.run(CoreProgressManager.java:208)
        at com.intellij.openapi.progress.impl.CoreProgressManager$TaskRunnable.run(CoreProgressManager.java:563)
        at com.intellij.openapi.progress.impl.CoreProgressManager$8.run(CoreProgressManager.java:367)
        at com.intellij.openapi.progress.impl.CoreProgressManager$2.run(CoreProgressManager.java:152)
        at com.intellij.openapi.progress.impl.CoreProgressManager.a(CoreProgressManager.java:452)
        at com.intellij.openapi.progress.impl.CoreProgressManager.executeProcessUnderProgress(CoreProgressManager.java:402)
        at com.intellij.openapi.progress.impl.ProgressManagerImpl.executeProcessUnderProgress(ProgressManagerImpl.java:54)
        at com.intellij.openapi.progress.impl.CoreProgressManager.runProcess(CoreProgressManager.java:137)
        at com.intellij.openapi.application.impl.ApplicationImpl$10$1.run(ApplicationImpl.java:623)
        at com.intellij.openapi.application.impl.ApplicationImpl$8.run(ApplicationImpl.java:405)
        at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:471)
        at java.util.concurrent.FutureTask.run(FutureTask.java:262)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
        at java.lang.Thread.run(Thread.java:745)
        at org.jetbrains.ide.PooledThreadExecutor$1$1.run(PooledThreadExecutor.java:56)
Caused by: com.intellij.diagnostic.PluginException: org/infernus/idea/checkstyle/CheckStylePlugin : Unsupported major.minor version 52.0 [Plugin: CheckStyle-IDEA] [Plugin: CheckStyle-IDEA]
        ... 28 more
Caused by: com.intellij.diagnostic.PluginException: org/infernus/idea/checkstyle/CheckStylePlugin : Unsupported major.minor version 52.0 [Plugin: CheckStyle-IDEA]
        at com.intellij.ide.plugins.cl.PluginClassLoader.b(PluginClassLoader.java:130)
        at com.intellij.ide.plugins.cl.PluginClassLoader.a(PluginClassLoader.java:77)
        at com.intellij.ide.plugins.cl.PluginClassLoader.loadClass(PluginClassLoader.java:66)
        at java.lang.ClassLoader.loadClass(ClassLoader.java:358)
        at java.lang.Class.forName0(Native Method)
        at java.lang.Class.forName(Class.java:274)
        at com.intellij.openapi.components.impl.ComponentManagerImpl$ComponentsRegistry.a(ComponentManagerImpl.java:427)
        ... 25 more
Caused by: java.lang.UnsupportedClassVersionError: org/infernus/idea/checkstyle/CheckStylePlugin : Unsupported major.minor version 52.0
        at java.lang.ClassLoader.defineClass1(Native Method)
        at java.lang.ClassLoader.defineClass(ClassLoader.java:800)
        at java.lang.ClassLoader.defineClass(ClassLoader.java:643)
        at com.intellij.util.lang.UrlClassLoader._defineClass(UrlClassLoader.java:259)
        at com.intellij.util.lang.UrlClassLoader.defineClass(UrlClassLoader.java:255)
        at com.intellij.util.lang.UrlClassLoader._findClass(UrlClassLoader.java:231)
        at com.intellij.ide.plugins.cl.PluginClassLoader.b(PluginClassLoader.java:124)
        ... 31 more

**Comment by jshiell** [comment]
Dup of #142. It's also documented in the release notes :smile:

TL;DR run IDEA on JDK 8. If you're on OS X, you'll either want to hack the app manifest, or use https://download.jetbrains.com/idea/ideaIU-14.1.1-custom-jdk-bundled.dmg
